mlflow/langchain/utils/chat.py

```diff
@@ -215,16 +242,16 @@ def _convert(chunk):
     return map(_convert, chunk_iter)
 
 
-def _convert_chat_request_or_throw(chat_request: dict):
+def _convert_chat_request_or_throw(chat_request: dict[str, Any]) -> list[Union[BaseMessage]]:
     if IS_PYDANTIC_V2_OR_NEWER:
-        model = _ChatRequest.model_validate(chat_request)
+        model = ChatCompletionRequest.model_validate(chat_request)
     else:
-        model = _ChatRequest.parse_obj(chat_request)
+        model = ChatCompletionRequest.parse_obj(chat_request)
```

Comment:
nit: i think we can replace this with the `BaseModel.validate_compat()` function `ChatCompletionRequest.validate_compat(chat_request)`

https://github.com/mlflow/mlflow/blob/d5b19f0628ff5d55a71648abd65ce95c7c1d0a47/mlflow/types/chat.py#L12-L21